### PR TITLE
feat: get cognito access_token programmatically 

### PR DESCRIPTION
**Pull request by nguyen102**
Issue #, if available:

Description of changes:
* Updated Cognito to allow grabbing cognito tokens with a script
* Added a script to get cognito tokens
* Update integration tests to use `access_tokens` for Auth
* Update integration tests to make environment requests against fake env with a `uuid` because using non-uuid will result in a "403" Auth error. Our current Auth framework  checks that each environment name is a UUID or it'll return a 403 error. 

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/integration-tests/support/setup.ts [hunk 1]
```diff
@@ -27,17 +29,30 @@ export default class Setup {
...
-
-    return session;
+    throw new Error('Implement createAdminSession');
   }
 
   public async getDefaultAdminSession(): Promise<ClientSession> {
     // TODO: Handle token expiration and getting defaultAdminSession instead of creating a new Admin Session
     if (this._defaultAdminSession === undefined) {
-      this._defaultAdminSession = await this.createAdminSession();
+      // this._defaultAdminSession = await this.createAdminSession();
```
Comment (SanketD92) [inline solutions/swb-reference/integration-tests/support/setup.ts:38]: remove commented code

File: workbench-core/base/src/cognitoTokenService.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    rootUsername: string,
+    rootPasswordParamStorePath?: string,
+    rootPassword?: string
+  ): Promise<{
+    accessToken: string;
+    idToken: string;
+    refreshToken: string;
+  }> {
+    let password: string = rootPassword || '';
+    if (rootPasswordParamStorePath && rootPassword) {
```
Comment (SanketD92) [inline workbench-core/base/src/cognitoTokenService.ts:28]: throw error if `rootPassword` is defined, irrespective of `rootPasswordParamStorePath`

Comment (SanketD92) [inline workbench-core/base/src/cognitoTokenService.ts:28]: Also throw if both are undefined.

File: solutions/swb-reference/integration-tests/support/setup.ts (hunk 1 above)
Comment (nguyen102) [reply solutions/swb-reference/integration-tests/support/setup.ts:38]: updated

File: workbench-core/base/src/cognitoTokenService.ts (hunk 2 above)
Comment (nguyen102) [reply workbench-core/base/src/cognitoTokenService.ts:28]: updated

**Review by SanketD92 — DISMISSED** [review]

**Review by bhudnell — APPROVED** [review]

**Pull request merged**